care/utils/serializer/phonenumber_ispossible_field.py

```diff
@@ -0,0 +1,18 @@
+import phonenumbers
+from django.utils.translation import gettext_lazy as _
+from phonenumber_field.phonenumber import to_python
+from rest_framework import serializers
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'rest_framework'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5343021)